person.rb

```diff
@@ -0,0 +1,25 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..10_000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if is_of_age? && @parent_permission
+
+    false
+  end
```

Comment:
- [ ] I think we should rethink this method. Requirements: `returns true if a person is of age or if they have permission from parents.`. However, the way you wrote will return false if parent_permission is true, but the person is not of age.